Blaise-Shyaka/Politico
# [#171720894] Create API endpoint to delete a political party

# What does this PR do?

It creates an endpoint that allows an admin to delete a political party

# Description of the tasks to be completed

- Write tests for the `DELETE /parties/:partyId` endpoint
- Write a controller for that endpoint
- Ensure that all tests pass

# How can be this be manually tested?

- Clone this repository
- In your terminal, run `npm install` to install the dependencies and `npm run test` to run tests
- All tests should pass

# What are the relevant pivotal tracker stories?

#171720894
